src/main/fc/config.c

```diff
@@ -72,6 +73,8 @@
 
 static bool configIsDirty; /* someone indicated that the config is modified and it is not yet saved */
 
+bool rebootRequired = false;  // set if a config change requires a reboot to take effect
```

Comment:
I'd probably make this static, and add a `getRebootRequired()` getter to avoid the pitfall of it directly being set, and then not working.

Reply:
Changed to a static and added getter function.